data/assetUpgraderFees.ts

```diff
@@ -0,0 +1,8 @@
+import {BigNumber} from '@ethersproject/bignumber';
+
+function sandWei(v: number) {
+  return BigNumber.from(v).mul("1000000000000000000").toString();
+}
+
+export const upgradeFee = sandWei(300);
+export const gemAdditionFee = sandWei(100);
```

Comment:
note that these numbers are made-up; they will need to be set correctly before deployment.